class ZealotError(Exception):
    pass


class NPlusOneError(ZealotError):
    pass
